### PR TITLE
fix(investigation table) 

### PR DESCRIPTION
**Pull request by sandymash**
- made comment tooltip disappear on scroll

### CODE REVIEW

**Review by eladyashar — APPROVED** [review]

**Review by daniel7764 — DISMISSED** [review]

**Review by ItayBenMoshe — APPROVED** [review]

**Review by nevobonneh — DISMISSED** [review]

**Comment by NaamanTsur** [comment]
I found that popperjs has a `hide` property so we can hide the popper when getting out of bounds

**Comment by NaamanTsur** [comment]
https://popper.js.org/docs/v1/#modifiershide

**Comment by NaamanTsur** [comment]
Closing it on scroll is fine too though

File: client/src/components/App/Content/LandingPage/InvestigationTable/commentDisplay/commentDisplay.tsx [hunk 1]
```diff
@@ -6,14 +6,20 @@ import useStyles from './commentDisplayStyles';
...
     const handleTooltipClose = () => setIsTooltipOpen(false);
     const handleTooltipToggle = (event:any) => {
         event.stopPropagation();
         setIsTooltipOpen(isOpen => !isOpen);
     };
 
+    React.useEffect(() => {
+        scrollableRef?.addEventListener('scroll', handleTooltipClose)
+        return () => scrollableRef?.removeEventListener('scroll', handleTooltipClose)
+    });
```
Comment (NaamanTsur) [inline client/src/components/App/Content/LandingPage/InvestigationTable/commentDisplay/commentDisplay.tsx:21]: I think you should add a useEffect condition only when the ref changes.

**Review by NaamanTsur — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/LandingPage/InvestigationTable/commentDisplay/commentDisplay.tsx (hunk 1 above)
Comment (NaamanTsur) [reply client/src/components/App/Content/LandingPage/InvestigationTable/commentDisplay/commentDisplay.tsx:21]: If i understand this correctly a new event listener will be created on each render

Comment (sandymash) [reply client/src/components/App/Content/LandingPage/InvestigationTable/commentDisplay/commentDisplay.tsx:21]: 👍  yup definitely.

**Comment by sandymash** [comment]
> I found that popperjs has a `hide` property so we can hide the popper when getting out of bounds

tries this one out when working on the initial us... didnt work

**Comment by NaamanTsur** [comment]
@sandymash Yeah you need to define a css selector referencing x-out-of-boundaries.

**Review by NaamanTsur — APPROVED** [review]

**Review by nevobonneh — APPROVED** [review]

**Pull request merged**